kpops/pipeline_generator/pipeline.py

```diff
@@ -180,6 +253,14 @@ def load_from_yaml(
 
         return cls(main_content, env_content, registry, config, handlers)
 
+    async def build_execution_graph_from(
+        self,
+        components: list[PipelineComponent],
+        reverse: bool,
+        runner: Callable[[PipelineComponent], Coroutine],
+    ) -> Awaitable:
+        return self.components.build_execution_graph_from(components, reverse, runner)
+
```

Comment:
This method is calling another method. Can't we just use the other one instead and get rid of this?

Reply:
Probably, but what do you recommend ? Calling `pipeline.components.build_execution_graph_from` outside?